common/djangoapps/student/views.py

```diff
@@ -1804,6 +1805,28 @@ def create_account_with_params(request, params):
     return new_user
 
 
+def _enroll_user_in_pending_courses(user_id):
+    """
+    Enroll student in any pending courses he/she may have.
+    """
+    student = User.objects.filter(id=user_id)
```

Comment:
The user is created in the `_do_create_account` call on L1635. In what scenario would the user not exist?